src/tsconfig.server.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../out-tsc/app",
+    "baseUrl": "./",
+    // Set the module format to "commonjs":
```

Comment:
Comment in JSON?

Reply:
I looked this up, and tsconfig files support comments. https://github.com/lacolaco/tsconfig-cli/pull/1

Reply:
Cool!